OpenShift/helm/nexus-repository-manager/README.md

```diff
@@ -118,7 +118,6 @@ The following table lists the configurable parameters of the Nexus chart and the
 | `nexus.resources`                           | Nexus resource requests and limits  | `{}`                                    |
 | `nexus.dockerPort`                          | Port to access docker               | `5003`                                  |
 | `nexus.nexusPort`                           | Internal port for Nexus service     | `8081`                                  |
-| `nexus.service.type`                        | Service for Nexus                   |`NodePort`                                |
```

Comment:
nothing is successfully reading the service.type, so drop it out.

Reply:
????